<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateOceanDiagForm</class>
 <widget class="QWidget" name="CreateOceanDiagForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>889</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Список станций с данными</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Пункт:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="StationWidget" name="point">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="isSearchVisible" stdset="0">
         <bool>false</bool>
        </property>
        <property name="isSavedStationsVisible" stdset="0">
         <bool>false</bool>
        </property>
        <property name="isHorisontal" stdset="0">
         <bool>false</bool>
        </property>
        <property name="isFilterVisible" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Срок: </string>
          </property>
          <property name="buddy">
           <cstring>departureTime</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="departureTime">
          <property name="displayFormat">
           <string>HH:mm</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="departureDate">
          <property name="displayFormat">
           <string>&quot;dd&quot; MMMM yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Угол выхода звукового луча:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="angleSpin"/>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="stationCount">
          <property name="text">
           <string>Всего станций: 0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="spaceSrcPlaceTree">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>50</number>
        </attribute>
        <column>
         <property name="text">
          <string>Название</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Широта</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Долгота</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Кол-во</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Глубина, м</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Расстояние, км</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>id</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StationWidget</class>
   <extends>QWidget</extends>
   <header location="global">meteo/commons/ui/custom/stationwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
